content/en/blog/2024/prom-and-otel/index.md

```diff
@@ -0,0 +1,564 @@
+of vendor- and community-built Prometheus exporters to aggregate and import data 
+into Prometheus.
+
+While you can use Prometheus to monitor a variety of infrastructure and 
+application metrics, one of its most popular use cases is to monitor Kubernetes. 
+This is the aspect of Prometheus monitoring that we will focus on in this 
+article.
+
+## Prometheus Metrics with OpenTelemetry
+In this section, you will learn about a couple of OTel collector components that 
+demonstrate the interoperability between OTel and Prometheus.  
+
+First, let’s do a quick refresher on the [collector](https://opentelemetry.io/docs/collector/) 
+– it’s an OTel component that can be used to collect and export data to and from 
+multiple sources and destinations. It also handles additional telemetry 
+processing, such as modifying data attributes and scrubbing personally 
+identifiable information. For example, you can use Prometheus SDKs to generate 
+metrics, ingest them with the collector, do some processing (if desired) and 
+then forward them to your chosen backend. 
+
+![Diagram showing the different components of the OTel collector](OTel-collector-refresher.png)
+
+The [Prometheus receiver](https://opentelemetry.io/docs/kubernetes/collector/components/#prometheus-receiver) allows you to collect metrics from any software that 
+exposes Prometheus metrics. It serves as a drop-in replacement for Prometheus to 
+scrape your services, and supports the [full set](https://github.com/prometheus/prometheus/blob/v2.28.1/docs/configuration/configuration.md#scrape_config) of configurations in 
+`scrape_config`. 
+
+If you are interested in [exemplars](https://opentelemetry.io/docs/specs/otel/metrics/data-model/#exemplars), which is a recorded value that associates OTel context to a metric 
+event, you can also use the Prometheus receiver to ingest them in the Prometheus 
+format and convert it to OTLP format. This enables you to correlate traces with 
+metrics. 
+
+Something to consider with this component is that it is under active 
+development; as such, it has several [limitations](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/prometheusreceiver/README.md#%EF%B8%8F-warning), including that it’s a stateful component. Additionally, it is recommended to not 
+use this component when multiple replicas of the collector are run, because in 
+this state: 
+* The collector is unable to auto-scale the scraping 
+* If the replicas are running with the same config, it will scrape the targets 
+multiple times
+* You will need to configure each replica with a different scraping config if 
+you want to manually shard the scraping
+
+The [Prometheus exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/prometheusexporter#prometheus-exporter) allows you to ship data in the 
+Prometheus format, which can then be scraped by a Prometheus server. It is used 
```

Comment:
I think the wording "Prometheus server" here confirms my thinking at the beginning of the document, i.e. that "server" is the right term rather than "tool".